build.sh

```diff
@@ -1,8 +0,0 @@
-#!/bin/bash
-
-set -e
-
-source .env
-
-docker build -t elifesciences/metypeset -t elifesciences/metypeset:${IMAGE_TAG} \
```

Comment:
The tag `elifesciences/metypeset` won't be changed by docker-compose, which is probably for good